lib/src/usage_tracking/model.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Ditto

Reply:
Yes, but the original was created in 2023.